### PR TITLE
Added bkeymaps page from old wiki

### PR DESCRIPTION
**Pull request by ectlunya**

### CODE REVIEW

File: software/bkeymaps.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+SETTING THE KEYBOARD LAYOUT
+________________________________________________________________________________
+
+
+Acquiring keymaps from the bkeymaps package
+________________________________________________________________________________
+
+You can get the bkeymaps package from Community.
```
Comment (dylanaraps) [inline software/bkeymaps.txt:8]:
```
The bkeymaps package is available in the Community repository.  
```

File: software/bkeymaps.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ kiss b bkeymaps                                                          |
+|   $ kiss i bkeymaps                                                          |
+|                                                                              |
++------------------------------------------------------------------------------+
```
Comment (dylanaraps) [inline software/bkeymaps.txt:15]:
```
Begin by first verifying that you have bkeymaps installed:

+------------------------------------------------------------------------------+
|                                                                              |
|   $ kiss b dhcpcd && kiss i dhcpcd                                           |
|                                                                              |
+------------------------------------------------------------------------------+
```

File: software/bkeymaps.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Now you can find the keymaps in /usr/share/bkeymaps.
```
Comment (dylanaraps) [inline software/bkeymaps.txt:17]: `The keymaps are located in /usr/share/bkeymaps.`

File: software/bkeymaps.txt [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+Loading Keymaps at login
+________________________________________________________________________________
+
+To load a keymap at login, you can add the command above to your .profile file.
```
Comment (dylanaraps) [inline software/bkeymaps.txt:23]:
```
To load a keymap at login, add the following command to your .profile
```

**Comment by dylanaraps** [comment]
You may find this template handy: https://github.com/kisslinux/wiki/blob/master/help/page-template.txt

File: software/bkeymaps.txt (hunk 2 above)
Comment (ectlunya) [reply software/bkeymaps.txt:15]: How about "First, install the bkeymaps package from the Community repository" to combine it with the previous comment?

Comment (dylanaraps) [reply software/bkeymaps.txt:15]:
```
Begin by first verifying that you have bkeymaps installed from the Community repository:
```

Comment (ectlunya) [reply software/bkeymaps.txt]: Isn't "Begin by first" redundant?

File: software/bkeymaps.txt [hunk 5]
```diff
@@ -0,0 +1,37 @@
+BKEYMAPS [0]
+________________________________________________________________________________
+
+Binary keymaps for busybox
```
Comment (dylanaraps) [inline software/bkeymaps.txt:4]: `loadkeys` (from `kbd`) also supports these keymap files: https://jlk.fjfi.cvut.cz/arch/manpages/man/loadkeys.1 

They're not only for busybox in other words.

File: software/bkeymaps.txt [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Load Keymaps at login
+________________________________________________________________________________
+
+To load a keymap at login, add the following command to your .profile
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ loadkmap < file                                                          |
+|                                                                              |
++------------------------------------------------------------------------------+
```
Comment (dylanaraps) [inline software/bkeymaps.txt:32]: Haven't tested `loadkeys` though something like this is probably better:

```

+------------------------------------------------------------------------------+
| Load keymap using busybox's loadkmap.                                        |
+------------------------------------------------------------------------------+
|                                                                              |
|   $ loadkmap < file                                                          |
|                                                                              |
+------------------------------------------------------------------------------+
| Load keymap using kbd's loadkeys.                                            |
+------------------------------------------------------------------------------+
|                                                                              |
|   $ loadkeys file                                                            |
|                                                                              |
+------------------------------------------------------------------------------+
```

File: software/bkeymaps.txt (hunk 2 above)
Comment (dylanaraps) [reply software/bkeymaps.txt]: :+1:

File: software/bkeymaps.txt [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+The keymaps will now be located in /usr/share/bkeymaps.
+
+
+Load Keymaps at login
+________________________________________________________________________________
+
+To load a keymap at login, add the following command to your .profile
```
Comment (dylanaraps) [inline software/bkeymaps.txt:26]: Add a full stop `.`.

File: software/bkeymaps.txt [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+Binary keymaps to set the keyboard layout
+
+
+Install keymaps
+________________________________________________________________________________
+
+
+Begin by verifying that you have bkeymaps installed from the Community
+repository
```
Comment (dylanaraps) [inline software/bkeymaps.txt:12]: Add a full stop `.`.

File: software/bkeymaps.txt [hunk 9]
```diff
@@ -0,0 +1,46 @@
+BKEYMAPS [0]
+________________________________________________________________________________
+
+Binary keymaps to set the keyboard layout
```
Comment (dylanaraps) [inline software/bkeymaps.txt:4]: Add a full stop `.`.

**Comment by dylanaraps** [comment]
I don't think `/software/` is the right place for this (maybe `/console/`?).  It might be better to rename the page to `keyboard-layout`. So `@/console/keyboard-layout`? Maybe even `tty/keyboard-layout`.
What do you think?

**Comment by ectlunya** [comment]
console sounds good. We can add the page about console fonts there too.

**Comment by ectlunya** [comment]
Are there any new console fonts to add to the list?

**Comment by dylanaraps** [comment]
bkeymaps.txt exists twice in this PR.

File: console/console-font.txt [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+Currently packaged Linux console fonts.
+
+- terminus-font (Community).
+
+
+Changing the Console Font
+________________________________________________________________________________
+
+Use the following command to change the font with your preferred font instead
```
Comment (dylanaraps) [inline console/console-font.txt:19]:
```
Use the following command to change the font with your preferred font instead
of FONT_NAME.
```

```
Use the following command to set the console font.
```

**Pull request merged**